text/0000-channel-participation-api-without-system-channel.md

```diff
@@ -0,0 +1,536 @@
+- This will allow OSNs to accept "List" commands and "Delete" of the system channel only. 
+- Delete the system channel from all OSNs.
+- OSNs will now accept the full set of channel participation commands.
+- It is the responsibility of all local orderer channel participation admins to delete the system channel from all OSNs.
+
+### Creating a system channel using the channel participation API
+- Start one or more OSNs without any channels. 
+- These OSNs should be configured to accept channel participation commands from their respective local orderer channel 
+  participation admins. This is achieved by setting the respective fields in the orderer.yaml file of each OSN.
+- These OSNs may belong to different organizations.
+- Generate the genesis block of a system channel. (The system channel contains the `Consortiums` config group, whereas 
+  application channels do not).
+- Create the system channel by invoking the "Join" command as described above, but supply the genesis block of the 
+  system channel, rather than that of an application channel.
+- The channel participation command "Join" will no longer accept commands, and channel creation can only 
+  be done using a config transaction on the system channel.
+- The channel participation commands "List" and "Delete" (of system channel only) will continue to function.
+
+# Reference-level explanation
+[reference-level-explanation]: #reference-level-explanation
+
+The implementation of the channel participation API can be split in two. One part deals with the "API layer" and 
+deals with exposing the API via an HTTP server. The second part receives abstract channel participation commands 
+via a Go interface, and executed the operations described above.  
+  
+## The API layer
+The peer and the orderer host an HTTP server that offers a RESTful “operations” API (see: 
+[The Operations Service](https://hyperledger-fabric.readthedocs.io/en/release-2.0/operations_service.html)).
+We intend to hook into the HTTP server that is hosted by the orderer, and expose a new capability 
+`/channel-participation`, which has a single resource  `/channels` as described above. 
+The channel participation commands will only be enabled if specified explicitly in the `orderer.yaml` 
+configuration file. These commands will only be available in the orderer and not in the peer.
+
+The _local orderer participation admin_  role will have the same authentication and authorization
+as the _operations admin_ (i.e. the same mutual TLS certificate). We choose this option because it is the simplest 
+to implement at this time. This is expected to change in the future (see below).
+
+## Participation management
+We intend to implement the channel participation management functionality in a new package 
+(`orderer/common/channelparticipation`). In this package a new class will keep a map of channel names and their status,
```

Comment:
This follows the existing patterns of the orderer codebase, but we have been moving towards placing new code not to be consumed by external parties as a library (though this is essentially all of Fabric) into `./internal`.  I'd welcome opinions from other maintainers, but either way, I don't see the location of this package as being significant enough to hold up the RFC.